proc_parent.go

```diff
@@ -390,6 +391,7 @@ func (mp *parent) fork() error {
 	//wait....
 	select {
 	case err := <-cmdwait:
+		mp.childCmdMut.Unlock()
```

Comment:
question: Since you removed the `defer mp.childCmMut.Unlock()` and added it to this case would you also need to include it in the following case after line 426?

Reply:
Thanks. Used sync.Once to add the defer back.